Implement onboarding review (#551)
returntocorp/semgrep-docs: docs/getting-started.md

- See [CLI Reference](../cli-reference/) for command line options and exit codes.
- Visit [Running rules](../running-rules/) to learn more or try Semgrep on known vulnerable test projects:

> - Explore Semgrep [rules](https://semgrep.dev/r) collection. 

Reviewer: ```suggestion
- Explore Semgrep [rules](https://semgrep.dev/r) collection to add rules to your project or try your hand at [writing a rule](../writing-rules/overview/) yourself. 
```

> - Explore Semgrep [rules](https://semgrep.dev/r) collection to add rules to your project or try your hand at [writing a rule](../writing-rules/overview/) yourself. 

Reviewer 2: ```suggestion
- Explore the Semgrep [Rule Registry](https://semgrep.dev/r) to add rules to your project or try your hand at [writing a rule](../writing-rules/overview/) yourself. 
```

</p>
</details>

## Run Semgrep continuously

> Semgrep is at its best when used to continuously scan code. Check out [Semgrep CI](../semgrep-ci/overview/) to learn how to get results where you already work: GitHub, GitLab, Slack, Jira, and more. To get results even earlier in the development process, such as in a Git pre-commit hook or VS Code, check the available [Semgrep extensions](../extensions/).

Reviewer 2: ```suggestion
Semgrep is at its best when used to continuously scan code. Check out [Semgrep in CI](../semgrep-ci/overview/) to learn how to get results where you already work: GitHub, GitLab, Slack, Jira, and more. To get results even earlier in the development process, such as in a Git pre-commit hook or VS Code, check the available [Semgrep extensions](../extensions/).
```
